src/app/auth/userlogin/userlogin.component.html

```diff
@@ -1,41 +1,74 @@
+        if (client_host === 'srt-client.app.cloud.gov' || client_host === 'srt.app.cloud.gov') { server_host = 'srt-server.app.cloud.gov'; }
+        if (client_host === 'srt-client-dev.app.cloud.gov') { server_host = 'srt-server-dev.app.cloud.gov'; }
+        if (client_host === 'srt-client-staging.app.cloud.gov') { server_host = 'srt-server-staging.app.cloud.gov'; }
+        if (server_host === '') { server_host = 'localhost:3000'; }
+        window.location = 'http://' + server_host + '/api/login';
+      }())">
+        <img src="assets/login-gov-logo.svg" alt="Login.gov">
+      </a>
     </div>
 
-    <div class="display-flex flex-column flex-align-center">
-      <button *ngIf="is_dev" 
-              class="usa-button usa-button--outline margin-2"
-              id="login-dev"
-              aria-label="Login as Dev"
-              onClick=" (function() {
-                        var client_host = window.location.hostname;
-                        var server_host = '';
-                        if (client_host === 'localhost') { server_host = 'localhost:3000'; }
-                        if (server_host === '') { server_host = 'localhost:3000'; }
-                        window.location = 'http://' + server_host + '/api/casLogin';
-                      }()) ">
-        Login as DEV
-      </button>
+    <!-- Spacer div to add padding between the buttons -->
+    <div class="spacer"></div>
 
-      <button class="usa-button usa-button--outline"
-              id="login-using-login-gov"
-              aria-label="Login using Login.gov"
-              onClick=" (function() {
-                        var client_host = window.location.hostname;
-                        var server_host = '';
-                        if (client_host === 'localhost') { server_host = 'localhost:3000'; }
-                        if (client_host === 'srt-client.app.cloud.gov' || client_host === 'srt.app.cloud.gov') { server_host = 'srt-server.app.cloud.gov'; }
-                        if (client_host === 'srt-client-dev.app.cloud.gov') { server_host = 'srt-server-dev.app.cloud.gov'; }
-                        if (client_host === 'srt-client-staging.app.cloud.gov') { server_host = 'srt-server-staging.app.cloud.gov'; }
-                        if (server_host === '') { server_host = 'localhost:3000'; }
-                        window.location = 'http://' + server_host + '/api/login';
-                      }()) ">
-        Login using Login.gov
-      </button>
-    </div>
+    <button class="usa-button dev-button" aria-label="Login as Dev" onclick="(function() {
+      var client_host = window.location.hostname;
+      var server_host = '';
+      if (client_host === 'localhost') { server_host = 'localhost:3000'; }
+      if (server_host === '') { server_host = 'localhost:3000'; }
+      window.location = 'http://' + server_host + '/api/casLogin';
+    }())">
+      Login as DEV
+    </button>
   </div>
-</main>
+
+  <!-- Styles moved to the bottom of the page -->
```

Comment:
Why are the styles here and not in the userlogin.component.scss file?

Reply:
moving too fast! I'll get this in the proper file.